frame/curve-amm/src/lib.rs

```diff
@@ -824,6 +769,73 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		pub fn create_pool(
+			who: &T::AccountId,
+			assets: Vec<T::AssetId>,
+			amplification_coefficient: FixedU128,
+			fee: Permill,
+			admin_fee: Permill,
+		) -> Result<T::PoolId, DispatchError> {
+			// Assets related checks
+			ensure!(assets.len() > 1, Error::<T>::NotEnoughAssets);
+			let unique_assets = BTreeSet::<T::AssetId>::from_iter(assets.iter().copied());
```

Comment:
is unique_assets sorted? to avoid address to create same pool with different order?

Reply:
could sort assets and hash. can check if such pools exists from specific wallet. may be.

Reply:
it is based on BTreeSet so it will prevent case that you are concerned with.